llama-index-core/llama_index/core/workflow/decorators.py

```diff
@@ -13,8 +13,9 @@ class StepConfig(BaseModel):
     accepted_events: List[Any]
     event_name: str
     return_types: List[Any]
-    pass_context: bool
+    context_parameter: Optional[str]
```

Comment:
Instead of passing a boolean flag to `step()`, we infer the request for a Context instance by the presence of a function parameter annotated with type `Context`

Reply:
Do we need to deprecate this param in the step decorator? Or what happens now when a user sets `pass_context=True` ? Just ignored and we rely on type introspection?

Reply:
Ah, I see further down its just a no-op, so non-breaking, nice!